ledger/participant-state/kvutils/tools/integrity-check/src/test/scala/ledger/participant/state/kvutils/tools/integritycheck/LogAppendingReadServiceFactorySpec.scala

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+
+package com.daml.ledger.participant.state.kvutils.tools.integritycheck
+
+import akka.actor.ActorSystem
+import akka.stream.Materializer
+import akka.stream.scaladsl.Sink
+import com.codahale.metrics.MetricRegistry
+import com.daml.ledger.validator.LedgerStateOperations
+import com.daml.metrics.Metrics
+import com.google.protobuf.ByteString
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.{AsyncFunSuite, AsyncWordSpec, Matchers}
+
+@RunWith(classOf[JUnitRunner])
+class LogAppendingReadServiceFactorySpec extends AsyncWordSpec with Matchers {
+  "LogFragmentingReadServiceFactory" should {
```

Comment:
This doesn't match the name of the test class (not sure whether that's on purpose or not).

Reply:
Definitely a mistake; fixed, thanks!